### PR TITLE
Include "last modified" in advanced filters

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/8869

## Testing
**URL to test:** 

Netlify [deploy with filters on](https://8869-last-modified--atd-moped-main.netlify.app/moped/projects?filter=eyI5NDVjMTMzMS1kNjFlLTQ4YmMtYjliOC03YTNmY2ZhMDZhMWIiOnsiaWQiOiI5NDVjMTMzMS1kNjFlLTQ4YmMtYjliOC03YTNmY2ZhMDZhMWIiLCJmaWVsZCI6InVwZGF0ZWRfYXQiLCJvcGVyYXRvciI6ImRhdGVfZ3JlYXRlcl90aGFuX2VxdWFsX3RvIiwiYXZhaWxhYmxlT3BlcmF0b3JzIjpbeyJvcGVyYXRvciI6Il9lcSIsImxhYmVsIjoiaXMiLCJkZXNjcmlwdGlvbiI6IlRoZSBkYXRlIGlzIGVxdWFsIHRvLiIsImVudmVsb3BlIjpudWxsLCJ0eXBlIjoiZGF0ZSIsImlkIjoiZGF0ZV9lcXVhbHMifSx7Im9wZXJhdG9yIjoiX2d0IiwibGFiZWwiOiJncmVhdGVyIHRoYW4iLCJkZXNjcmlwdGlvbiI6IlRoZSBkYXRlIGlzIGdyZWF0ZXIgdGhhbi4iLCJlbnZlbG9wZSI6bnVsbCwidHlwZSI6ImRhdGUiLCJpZCI6ImRhdGVfZ3JlYXRlcl90aGFuIn0seyJvcGVyYXRvciI6Il9ndGUiLCJsYWJlbCI6ImdyZWF0ZXIgdGhhbiBvciBlcXVhbCB0byIsImRlc2NyaXB0aW9uIjoiVGhlIGRhdGUgaXMgZ3JlYXRlciB0aGFuIG9yIGVxdWFsIHRvLiIsImVudmVsb3BlIjpudWxsLCJ0eXBlIjoiZGF0ZSIsImlkIjoiZGF0ZV9ncmVhdGVyX3RoYW5fZXF1YWxfdG8ifSx7Im9wZXJhdG9yIjoiX2x0IiwibGFiZWwiOiJsZXNzIHRoYW4iLCJkZXNjcmlwdGlvbiI6IlRoZSBkYXRlIGlzIGxlc3MgdGhhbi4iLCJlbnZlbG9wZSI6bnVsbCwidHlwZSI6ImRhdGUiLCJpZCI6ImRhdGVfbGVzc190aGFuIn0seyJvcGVyYXRvciI6Il9sdCIsImxhYmVsIjoibGVzcyB0aGFuIG9yIGVxdWFsIHRvIiwiZGVzY3JpcHRpb24iOiJUaGUgZGF0ZSBpcyBsZXNzIHRoYW4uIiwiZW52ZWxvcGUiOm51bGwsInR5cGUiOiJkYXRlIiwiaWQiOiJkYXRlX2xlc3NfdGhhbl9lcXVhbF90byJ9XSwiZ3FsT3BlcmF0b3IiOiJfZ3RlIiwiZW52ZWxvcGUiOm51bGwsInBsYWNlaG9sZGVyIjoiTGFzdCBtb2RpZmllZCIsInZhbHVlIjoiMjAyMi0wMy0xNSIsInR5cGUiOiJkYXRlIiwibGFiZWwiOiJMYXN0IG1vZGlmaWVkIn19)

**Steps to test:**

Open advanced filters, find "Last modified" as an option and try some dates.

I put it in the middle of the list of dropdown options, let me know if its better elsewhere. 

I also included an update to the padding on the signals dashboard that i noticed while testing, let me know if it should be spun out into a separate branch. 

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Review by frankhereford — APPROVED** [review]
Thanks for taking the time to talk about the `&` nesting CSS selector. Even though it was coming out in this PR, I appreciate your help getting more familiar with that feature. 

This PR is tested in place on Netlify and worked great and the code is reviewed too.

🚢 🚢

**Review by amenity — APPROVED** [review]
💯 🙌 Thanks, @chiaberry!

File: moped-editor/src/views/projects/signalProjectTable/SignalProjectTable.js [hunk 1]
```diff
@@ -34,12 +34,6 @@ import RenderSignalLink from "./RenderSignalLink";
 import TaskOrderAutocomplete from "./TaskOrderAutocomplete";
 
 const useStyles = makeStyles({
-  signalsTable: {
-    "& .MuiTableCell-root": {
-      // override the default padding of 16px
-      padding: "10px",
-    },
-  },
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/signalProjectTable/SignalProjectTable.js:42]: 👋 bye

**Review by mateoclarke — APPROVED** [review]
🚢 💨 ✨ 

filter-pretty

**Pull request merged**